packages/components/.storybook/native/storybook.requires.ts

```diff
@@ -1,6 +1,10 @@
 /* do not change this file, it is auto generated by storybook. */
```

Comment:
Highlighting that all changes in this file happened automatically either from the storybook update process or building after update.